Description: add format argument to sprintf call to prevent failure with -Werror=format-security
Origin: vendor
Bug-Debian: http://bugs.debian.org/bug=643390
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-12-19

--- a/barcode-0.98/plessey.c
+++ b/barcode-0.98/plessey.c
@@ -148,7 +148,7 @@
                         checkptr[i+j] ^= check[j];
     }
     for (i = 0; i < 8; i++) {
-        sprintf(ptr, patterns[checkptr[strlen(text) * 4 + i]]);
+        sprintf(ptr, "%s", patterns[checkptr[strlen(text) * 4 + i]]);
 	ptr += 2;
     }
     fprintf(stderr, "CRC: ");
